wire-library/wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WirePlugin.kt

```diff
@@ -90,6 +95,17 @@ class WirePlugin : Plugin<Project> {
       it.description = "Aggregation task which runs every generation task for every given source"
     }
 
+    if (extension.protoLibrary) {
+      val libraryProtoSources = File("${buildDir}/wire/proto-sources")
+      val sourceSets = project.extensions.getByType(SourceSetContainer::class.java)
+      sourceSets.getByName("main") { main: SourceSet ->
+        main.resources.srcDir(libraryProtoSources)
+      }
```

Comment:
I wonder if that would work on Android with variants, but I guess we don't expect Android modules to be proto libraries anyway.
I can check later to see what happens. We could just log a warning if `main` isn't found.